packages/web-react/src/components/TextFieldBase/TextFieldBase.tsx

```diff
@@ -13,6 +13,17 @@ export const TextFieldBase = (props: SpiritTextFieldBaseProps) => {
   const { id, label, message, ...restProps } = modifiedProps;
   const { styleProps, props: otherProps } = useStyleProps(restProps);
 
+  useDeprecationMessage({
+    method: 'property',
+    trigger: props?.validationState === 'error',
+    componentName: 'TextFieldBase',
```

Comment:
Be aware, the developer will use only TextField or TextArea component. So this component name is misleading. I suggest using the `isMultiline` prop to distinguish between TextField and TextArea and thus set a correct component name.

Reply:
done